explainers/navigation-to-unsigned-bundles.md

```diff
@@ -35,6 +29,47 @@ images or videos, users have native apps like
 
 <!-- /TOC -->
 
+## Goals
+
+1. Provide a way for users to share web content to other users who may be
+   offline when they try to visit the content. This should work with their
+   existing file-sharing apps, like [SHAREit](https://www.ushareit.com/),
+   [Xender](http://www.xender.com/), or [Google
+   Files](https://files.google.com/), which present the unit of sharing as a
+   single file.
+
+   <details>
+   <summary>
+
+   ![A drawing of a person downloading a site, bundling its content, and passing it to a friend without their own connection to the internet.](http://www.plantuml.com/plantuml/svg/NOonQiD044Jx_Oh91zWVO8nm4KAgj6dIdAILlN2tA_RsS17oxpbZ156wOURDkzH87grachAr6PyyLWd6Dm6BPCQQhdoyHSbR8UNHh7gb7r2QmXolKiDbR8zyFh-tadGOQ5CT3iEEE47DIyeOtUvLkYXvoD9Lk3ylnx7fd9d-lhfaltRFCrJwDtJqpOLr1ga58so5BLjtmeTXCbMUGao_D0nnOuW6ktAyqALZhUHV)
+
+   </summary>
+
+   ```plantuml
+   @startuml
+   cloud Website {
+     file page.html
+     file image.png
+   }
+   actor Distributor <<Human>>
+   Website --> Distributor : normal browsing
+   artifact website.bundle
+   Distributor -> website.bundle : bundles
+   website.bundle -> Friend
+   note top of Friend : No connection\nto the internet!
+   @enduml
+   ```
+
+   </details>
+
+1. Provide a way for sites like https://www.isocfoundation.org/ to package their
+   content so communities can fetch it once over a perhaps-expensive link and
+   then share it internally without each recipient needing to be online when
+   they get it.
+1. Provide a way for sites like https://archive.org/ to distribute archived
+   content without having to rewrite all its internal links.
+
```

Comment:
Maybe also say "see also the alternate formats section (link) to see why we are proposing this with a new format" or something (but up to you)

Reply:
Thanks, done.